eras/conway/impl/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 
 ### `testlib`
 
+* Added `ToExpr` instance for `ConwayNewEpochPredFailure`
```

Comment:
This actually no need to go into `1.16.1.0`, since we just made a release. Sorry :slightly_smiling_face:

Reply:
Actually, nevermind that instance got released in `1.16.0.0`